man/man8/zfs.8

```diff
@@ -983,6 +1016,73 @@ This property can also be referred to by its shortened column name,
 
 .sp
 .ne 2
+.na
+\fB\fBencryption\fR=\fBoff\fR | \fBon\fR | \fBaes-128-ccm\fR |
+\fBaes-192-ccm\fR | \fBaes-256-ccm\fR | \fBaes-128-gcm\fR |
+\fBaes-192-gcm\fR | \fBaes-256-gcm\fR
+.ad
+.sp .6
+.RS 4n
+Controls the encryption cipher suite (block cipher, key length, and mode) used
+for this dataset. Requires the encryption feature to be enabled on the pool.
+Requires a \fBkeyformat\fR to be set at dataset creation time.
+.sp
+Selecting \fBencryption\fR=\fBon\fR when creating a dataset indicates that the
+default encryption suite will be selected, which is currently \fBaes-256-ccm\fR.
+In order to provide consistent data protection, encryption must be specified at
+dataset creation time and it cannot be changed afterwards.
+.sp
+For more details and caveats about encryption see \fBzpool-features\fR.
+.RE
+
+.sp
+.ne 2
+.mk
+.na
+\fB\fBkeyformat\fR=\fBraw\fR | \fBhex\fR | \fBpassphrase\fR
+.ad
+.sp .6
+.RS 4n
+Controls what format the user's encryption key will be provided as. This 
+property is only set when the dataset is encrypted.
+.sp
+Raw keys and hex keys must be 32 bytes long (regardless of the chosen
+encryption suite) and must be randomly generated. Passphrases must be between
+8 and 64 bytes long and will be processed through PBKDF2 before being used
+(see the \fBpbkdf2iters\fR property). Even though the encryption suite cannot
+be changed after dataset creation, the keyformat can be with
+\fBzfs change-key\fR. 
+.RE
+
+.sp
+.ne 2
+.mk
+.na
+\fB\fBkeylocation\fR=\fBprompt\fR | \fBfile://\fR\fI<absolute file path\fR>
+.ad
+.sp .6
+.RS 4n
+Controls where the user's encryption key should be loaded from. This property
+is only set when the dataset is encrypted.
+.sp
+If \fBkeylocation\fR is set
+locally, the dataset is an encryption root. Encryption roots share their keys
+with all datasets that inherit this property. This means that when a key is
+loaded for the encryption root, all children that inherit the \fBkeylocation\fR
+property are automatically loaded as well.
+.sp
+Even though the encryption suite cannot be changed after dataset creation, the
+keylocation can be with \fBzfs change-key\fR. If the dataset is an encryption 
+root this property may also be changed with \fBzfs set\fR. If \fBprompt\fR is
```

Comment:
That is correct. non-encryption roots can use `zfs change-key`, which will make them an encryption root. I will add a sentence to the man page to reflect this.